### PR TITLE
Modified table view CSS per code review comments.

### PR DESCRIPTION
**Pull request by dlabrecq**
## Description

Adjusted CSS per Gabriel's code review comments from:

https://github.com/patternfly/patternfly/pull/461
## Changes
- Avoiding shorthand declarations.
- Commented out unused dataTables_wrapper class.
- Nested datatable-pf-actions class.
- Removed unused attributes: font-weight, height, line-height, vertical-align and width.
- Removed float:right from toolbar-pf-action-right override.
- Added padding for inline actions kebab.
- Renamed JS util objects.
## Link to rawgit and/or image

https://rawgit.com/dlabrecq/patternfly/table-view-dist/dist/tests/datatables.html

This is an image:

screen shot 2016-09-14 at 2 57 44 pm
## PR checklist (if relevant)
- [x] **Cross browser**: works in IE9
- [x] **Cross browser**: works in IE10
- [x] **Cross browser**: works in IE11
- [x] **Cross browser**: works in Edge
- [x] **Cross browser**: works in Chrome
- [x] **Cross browser**: works in Firefox
- [x] **Cross browser**: works in Safari
- [x] **Cross browser**: works in Opera
- [x] **Responsive**: works in extra small, small, medium and large view ports.
- [x] **Preview the PR**: An image or a URL for designer to preview this PR is provided.

### CODE REVIEW

**Comment by dlabrecq** [comment]
@cardosogabriel please review. Thanks!

File: less/datatables.less [hunk 1]
```diff
@@ -263,58 +264,73 @@
     }
   }
   .toolbar-pf {
-    background: @table-bg-accent;
-    border: 1px solid @table-border-color;
```
Comment (cardosogabriel) [inline less/datatables.less]: Avoiding shorthand here would be:
`border-width: 1px;
border-style: solid;
border-color: @table-border-color;`

File: less/datatables.less [hunk 2]
```diff
@@ -263,58 +264,73 @@
...
   }
 }
 
 .datatable-pf-actions {
-  background: @table-bg-accent;
+  background-color: @table-bg-accent;
   > .btn {
     border: none;
     box-shadow: none;
+    height: 100%;
```
Comment (cardosogabriel) [inline less/datatables.less]: Somehow there is a "margin" bottom that makes the button not be fully vertically centralized.
I believe if we make this button 100% height, the kebab should also be. This alignment is weird (action button in the middle, kebab at the top). Maybe u clarify with the designer.
table resized

**Comment by cardosogabriel** [comment]
@dlabrecq , thanks for the fixes, they look good. I only made 2 comments inline.

File: less/datatables.less (hunk 1 above)
Comment (dlabrecq) [reply less/datatables.less]: This has already been changed.

File: less/datatables.less (hunk 2 above)
Comment (dlabrecq) [reply less/datatables.less]: Modified so the button and kebab are both vertically aligned.

Comment (dlabrecq) [reply less/datatables.less]: screen shot 2016-09-26 at 3 01 35 pm

Comment (cardosogabriel) [reply less/datatables.less]: Thanks, looks better :)

File: less/datatables.less [hunk 3]
```diff
@@ -263,58 +264,73 @@
...
+        margin-left: -10px;
+        margin-right: -10px;
+      }
     }
     .toolbar-pf-action-right {
+      // Overriding "float: none" of toolbar-pf-action-right
+      @media (max-width: @grid-float-breakpoint) {
+        float: right;
+      }
       > .btn {
```
Comment (andresgalante) [inline less/datatables.less]: Is there a reason to use so many child selectors?

File: less/datatables.less [hunk 4]
```diff
@@ -263,58 +264,73 @@
...
-      border-top: 1px solid @table-border-color;
-      height: 40px;
-      line-height: 40px;
     }
   }
 }
 
 .datatable-pf-actions {
-  background: @table-bg-accent;
+  background-color: @table-bg-accent;
```
Comment (andresgalante) [inline less/datatables.less]: The actions btns on the right of each row are missing the gradient. That can easily be achieved by changing `btn-link` to `btn-default` then removing the borders.

**Comment by andresgalante** [comment]
Hi @dlabrecq I have some questions:
- Is there any reason to add a `container-fluid` for the toolbar?
- Do we need to float the the kebabs?

Thanks!

**Comment by dlabrecq** [comment]
@andresgalante I copied HTML from the toolbar example, which included container-fluid. Without container-fluid, the toolbar expands beyond the table because of the lack of padding. However, I can remove it and set the margin of .toolbar-pf to zero. Curious why the toolbar example uses container-fluid, but not here?

Regarding the kebabs, are you speaking about the pull-right selector? That was copied from an example, but will remove it.

File: less/datatables.less (hunk 4 above)
Comment (dlabrecq) [reply less/datatables.less]: Done.

File: less/datatables.less (hunk 3 above)
Comment (dlabrecq) [reply less/datatables.less]: Not here. I will remove it. There are areas where it's necessary due to specificity (e.g., the existing table sectors). I'll try to clean what I can.

**Comment by dlabrecq** [comment]
@andresgalante Can you please merge?

**Comment by bleathem** [comment]
@andresgalante and @cardosogabriel have both indicated they are happy with the state of this PR.  Nice work @dlabrecq!

Merging

**Pull request merged**